### PR TITLE
Add support for datasource resource group

### PR DESCRIPTION
**Pull request by hkantare**

### CODE REVIEW

File: ibm/data_source_ibm_resource_group_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+				Config: testAccCheckIBMResourceGroupDataSourceConfigWithName(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.ibm_resource_group.testacc_ds_resource_group_name", "name", "default"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMResourceGroupDataSourceConfigDefault() string {
```
Comment (Praveengostu) [inline ibm/data_source_ibm_resource_group_test.go]: Both are optional fields can you add a test case if none of them are specified

Comment (hkantare) [reply ibm/data_source_ibm_resource_group_test.go]: Added testcase to handle is_default false and validating the error.
